scripts/pluto_create_commission.rb

```diff
@@ -227,7 +227,7 @@ def call(severity, time, progname, msg)
 									workingGroupUID: workingGroupID,
 									client: $commissionClientName,
 									projectTypeUID: projectTypeID,
-									subscribingGroupIDs: '24',
+									subscribingGroupIDs: $subscribingGroupIDs,
 									ownerID: '10',
```

Comment:
Not sure of the context of this PR, but judging by the change, would the `ownerID` cause similar pains as it too is a magic number?